main.nf

```diff
@@ -32,9 +32,10 @@ def helpMessage() {
       --chromosome_size [file]                  Path to chromosome size file
       --restriction_fragments [file]            Path to restriction fragment file (bed)
       --save_reference [bool]                   Save reference genome to output folder. Default: False
-      --save_aligned_intermediates [bool]       Save intermediates alignment files. Default: False
 
     Alignments
+      --split_fastq [bool]                      Size of read chuncks to use to speed up the workflow. Default: None
```

Comment:
A bit confusing parameter description: It is set to `false` in `nextflow.config`, here it is a boolean with Default `None` and in the `nextflow_schema.json` it is a number (with no default).

Reply:
I updated the help

Reply:
Sorry, but I am still not satisfied with `split_fastq`. It is now a `int` in the help text in main.nf, with a `None`as default (not sure if this is considered an integer in groovy). It is still set as `false` (which should be the same as the default value in main.nf) and it is of type number in nextflow_schema.json without any default value. So, if you sync all these instances, I am fine with this PR.

Reply:
Yes, good point. Maybe you check for that value to not be `0` instead @nservant ? and update the defaults everywhere to `0` and the help in `main.nf`.